lib/mix/lib/mix/tasks/usage.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Mix.Tasks.Usage do
+  use Mix.Task
+
+  @shortdoc "Prints usage information for mix"
+
+  @moduledoc ~S"""
+  Usage: mix [task]
+    mix                       - Runs the default task (current: "mix run")
+    mix new                   - Creates a new Elixir project
+
+  For further help:
+    mix help                  - Prints help information for tasks
+    mix help                  - Prints all tasks and their shortdoc
+    mix help TASK             - Prints full docs for the given task
+    mix help --search PATTERN - Prints all tasks that contain PATTERN in the name
+    mix help --names          - Prints all task names and aliases
+    mix cmd                   - Executes the given command
+    mix compile               - Compiles source files
+    mix deps                  - Lists dependencies and their status
+    mix do                    - Executes the tasks separated by comma
+    mix escript               - Lists installed escripts
+    mix loadconfig            - Loads and persists the given configuration
+    mix local                 - Lists local tasks
+    mix local.hex             - Installs Hex locally
+    mix local.public_keys     - Manages public keys
+    mix local.rebar           - Installs Rebar locally
+    mix profile.cprof         - Profiles the given file or expression with cprof
+    mix profile.fprof         - Profiles the given file or expression with fprof
+    mix run                   - Runs the given file or expression
+    iex -S mix                - Starts IEx and runs the default task
+  """
```

Comment:
This will be hard to maintin. We will better load the list dinamically as in https://github.com/elixir-lang/elixir/blob/7d9072c729425fc54ccac316d1dc12801ee3373e/lib/mix/lib/mix/tasks/help.ex#L43